README.md

```diff
@@ -7,15 +7,16 @@
  - [x] Tribunal TGIF
  - [ ] Hiking
  - [x] The Social network x1
- - [ ] Brazilian Steakhouse (for a good reason)
+ - [ ] <s>Brazilian Steakhouse (for a good reason)</s>
  - [ ] Kings Island
- - [ ] 16 Bit
+ - [ ] <s>16 Bit</s>
  - [ ] <s>Dueling Piano Bar </s>
  - [ ] Nada
  - [ ] Short Vine Bar Hop - Hit every bar on shortvine from Kroger to Taste of Belgium
  - [x] Engineering Luau
  - [x] Buzzed Bull Creamery
  - [x] Ohio State Fair
  - [ ] Canoeing
- - [ ] Terry's Turf Club
+ - [ ] <s>Terry's Turf Club</s>
```

Comment:
??

Reply:
Terry’s Turf Club is canceled due to sexual harassment and not paying employees, or something like that.

Reply:
Looks like it's under new ownership and just "Turf Club". Not sure if it makes it worth not crossing out - we're pretty unlikely to do it. https://www.bizjournals.com/cincinnati/news/2019/01/09/new-owner-talks-about-what-will-change-stay-the.html

Reply:
Updated.